red_black_cards.py

```diff
@@ -0,0 +1,44 @@
+#Problem Statement: "You are given a standard deck of cards with 26 black cards and 26 red cards. You're now going to play a game: 
+#in every round, you can take one of two actions: you can stop playing, or you can draw another card. If you draw another card and 
+#it's red, you get one dollar. If you draw another card and it's black, you lose one dollar. What is the expected value of this game?"
+
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+    red = int(input('How many red cards are there?\n'))
+except:
+    print("Input value cannot be cast to an int")
+    exit()
+
+def expected_value(int: b, int: r) -> int:
+    "If either r or b is negative, then we cannot return a valid expected value"
+    if (r < 0 or b < 0):
+        print("Number of red and black cards must both be non-negative")
+        exit()
```

Comment:
Do not use calls to `exit`, and you won't need it -- if the condition is not satisfied, then you simply won't execute the rest of the code in this function